### PR TITLE
Moved ignore error to kwarg

### LINKED ISSUE #44: Make do_not_raise a keyword argument to create_x() instead of a class attribute
I often find myself confused about why some errors get swallowed. I think it would be more convenient to add a `exist_ok` flag for `create_x()` methods, similar to  `Path.mkdir`, to make it more intuitive. 

It would also make it easier for the CLI, where we want to tell the user that there has been a conflict

### LINKED ISSUE #52: ensure all packages included in MVG release
When new package is added, tests pass because MVG package is not installed locally and files are directly accessed but the package will not be available for the user that installs it.

Either:
1. Make sure the tests use a locall installation of package using `setup.py`
2. or, use `setuptools.find_packages` as described [here](https://packaging.python.org/tutorials/packaging-projects/#configuring-metadata) to automatically collect all packages

### PR DESCRIPTION
**Pull request by OskarLiew**
Now `MVG` doesn't automatically ignore all 409 errors without passing a flag to `create_*` method. Also fixed setup.py to always include all subpackages

Closes #52, Closes #44

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/vikinganalytics/mvg",
-    packages=["mvg", "mvg/features"],
+    packages=setuptools.find_packages(),
```
Comment (tuix) [inline setup.py:31]:
```suggestion
    packages=setuptools.find_packages(where="mvg"),
```
Better to limit to the `mvg` folder.

File: mvg/mvg.py [hunk 2]
```diff
@@ -257,17 +243,27 @@ def create_source(self, sid: str, meta: dict):
 
         meta : dict
             meta information
+
+        ignore_conflict : bool
```
Comment (tuix) [inline mvg/mvg.py:247]: The word conflict is coming from the title of HTTP error 409. From user's point of view this error happens when the source or measurement already exists. What about calling it `ignore_existing` or similar to `mkdir` call it `exist_ok`.

File: mvg/mvg.py [hunk 3]
```diff
@@ -257,17 +243,27 @@ def create_source(self, sid: str, meta: dict):
...
+            caused by trying to create an existing source. Defaults to False
         """
 
         logger.info("endpoint %s", self.endpoint)
         logger.info("creating source with source id=%s", sid)
         logger.info("metadata: %s", meta)
 
+        do_not_raise = []
+        if ignore_conflict:
+            do_not_raise.append(409)
```
Comment (tuix) [inline mvg/mvg.py:258]: A bit more expressive and self-documenting, maybe?
```suggestion
            do_not_raise.append(requests.codes.CONFLICT) # 409
```

**Review by tuix — CHANGES_REQUESTED** [review]

File: mvg/mvg.py (hunk 3 above)
Comment (OskarLiew) [reply mvg/mvg.py:258]: I agree, very nice suggestion

**Review by vnadhan — APPROVED** [review]

**Review by tuix — APPROVED** [review]

**Pull request merged**